src/open_samus_returns_rando/files/templates/randomizerpowerup.lua

```diff
@@ -182,7 +159,7 @@ function RandomizerPowerup.HandlePickupResources(progression, actorOrName)
                         end
                         Scenario.UpdateDNACounter()
                         RandomizerPowerup.IncreaseItemAmount("ITEM_ADN", resource.quantity)
-                        Game.AddSF(0, "RandomizerPowerup.ObjectiveComplete", "")
+                        RandomizerPowerup.ObjectiveComplete()
```

Comment:
I assume it's fine? Iirc multiworld encapsulates granting an item in a scheduled function anyways, so it shouldn't be a problem.

Reply:
I'm pretty sure the reason it was scheduled before was to actually show up after the item popup. But since the popups are asynchronous then it doesn't matter anymore.